### DISCUSSION TITLE
Bring all windows to front

### DISCUSSION THREAD

**Discussion by diminutivesloop** [rxhanson/Charmstone-Community#5] [Ideas]
It would be great if there was a way to mimic the behavior of the dock where clicking an app with multiple windows brings all windows to the front.

**Comment by rxhanson** [comment]
Unfortunately, the functionality provided in macOS that should allow this behavior doesn't work. It appears to be a bug Apple side. 

If Charmstone were not sandboxed & on the App Store, I could make a kind of hacky workaround for it based on window management APIs, but I have no plans to make a version of Charmstone that is not on the App Store at this time.
